src/content/ja/2021/seo.md

```diff
@@ -0,0 +1,758 @@
+
+{{ figure_markup(
+   image="image-loading-property-usage.png",
+   caption="画像読み込みのプロパティの使用状況。",
+   description='ページと画像読み込みプロパティの使用率（missing, lazy, eager, invalid, auto, blank）を示す棒グラフ。当社のデータでは、デスクトップページの83.3%、モバイルページの83.5%で画像の読み込みプロパティが欠落していることが判明しました。その結果、デスクトップとモバイルのページの15.6%が `loading="lazy"` を使用しており、`loading="eager"` はデスクトップとモバイルのページの0.8%に過ぎないことがわかりました。その他の画像読み込みプロパティである `loading="invalid"` や `loading="auto"` 、 `loading="blank"` は、デスクトップやモバイルのページの1%未満にしか使用されていません。',
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=1305654777&format=interactive",
+   sheets_gid="55531578",
+   sql_file="image-loading-property-usage.sql"
+   )
+}}
+
+私たちはそれを見たのです。
+
+- 85.5%のページでは、画像の `loading` プロパティが使用されていません。
+- 15.6%のページが `loading="lazy"` を使用しており、ビューポートに入る寸前まで画像の読み込みを遅らせています。
+- 0.8%のページでは、ブラウザがコードを読み込むと同時に画像を読み込む `loading="eager"` が使用されています。
+- 0.1%のページで無効なローディングプロパティが使用されています。
+- 0.1%のページでは、ブラウザのデフォルトの読み込み方法を使用する `loading="auto"` が使用されています。
+
+
+### 単語数
+
+ページ内の文字数はランキング要因ではありませんが、ページが文字を配信する方法はランキングに大きな影響を与えます。言葉は、未加工のページコードに含まれることも、レンダリングされたコンテンツに含まれることもあります。
+
+#### レンダリング文字数
+
+まず、レンダリングされたページの内容を見ます。_レンダリング_とは、ブラウザがすべてのJavaScriptとDOMまたはCSSOMを変更する他のコードを実行した後のページのコンテンツです。
+
+{{ figure_markup(
+   image="visible-rendered-words-percentile.png",
+   caption="レンダリングされた可視ワードの割合。",
+   description="パーセンタイル（10、25、50、75、90）別にレンダリングされた可視ワード数を示す棒グラフ。レンダリングされたデスクトップページの中央値は425語、モバイルページの中央値は367語でした。モバイル・ページで、すべてのパーセンタイルでレンダリングされる単語数が少なくなっています。",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=833732027&format=interactive",
+   sheets_gid="455169599",
+   sql_file="seo-stats-by-percentile.sql"
+   )
+}}
+
+- デスクトップページのレンダリング文字数の中央値は425語で、2020年には402語になっています。
+- モバイルページのレンダリングの中央値は367語であるのに対し、2020年には348語となっています。
+- モバイルページのレンダリングは、デスクトップページのレンダリングより13.6%少ない単語数です。Googleはモバイル専用のインデックスであることに注意してください。モバイル版にないコンテンツはインデックスされない可能性があります。
+
+#### 未加工のワード数
+
+次に、未加工のページコンテンツについて見ていきます。_未加工_とは、ブラウザがJavaScriptやDOMやCSSOMを修正する他のコードを実行する前のページのコンテンツのことです。ソースコードで配信され、目に見える「未加工」のコンテンツです。
+
+{{ figure_markup(
+   image="visible-raw-words-percentile.png",
+   caption="未加工の可視化単語の割合。",
+   description="パーセンタイル（10、25、50、75、90）ごとの未加工の可視ワード数を示す棒グラフ。デスクトップ用未加工ページの中央値は369語、モバイル用ページの中央値は321語でした。モバイル・ページで、すべてのパーセンタイルで未加工の単語数が少なくなっています。",
+   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQ7oSHa6DHYTfZqsuGmIYdlfYVkwrUvOYD_r6soecExV_ZpbbZjmG6watu0hwrOKqK3-inNrt0TfXCO/pubchart?oid=58186900&format=interactive",
+   sheets_gid="455169599",
+   sql_file="seo-stats-by-percentile.sql"
+   )
+}}
+
+- デスクトップページの中央値は369語であるのに対し、2020年には360語になっています。
+- モバイルページの中央値は321語であるのに対し、2020年には312語となっています。
+- モバイルの未加工ページは、デスクトップのレンダリングページよりも13.1%少ない単語しか含まれていません。Googleはモバイル専用のインデックスであることに注意してください。モバイルHTML版にないコンテンツは、インデックスされない可能性があります。
```

Comment:
Note in https://github.com/HTTPArchive/almanac.httparchive.org/pull/2788/files we corrected an error in this line. It looks like this error is still here. Can you correct?

Reply:
OK, I translated.